giddy/__init__.py

```diff
@@ -7,10 +7,10 @@
 
 """
 
-from . import directional
-from . import ergodic
-from . import markov
-from . import mobility
-from . import rank
-from . import util
-from . import sequence
+from . import directional  # noqa F401, E402
```

Comment:
what does # noqa F401, E402 mean?

Reply:
These are [codes telling `ruff`](https://docs.astral.sh/ruff/rules/#pyflakes-f) that those lines are not problems:

Code | Name | Message
-- | -- | --
F401 | unused-import | {name} imported but unused; consider using importlib.util.find_spec to test for availability
E402 | module-import-not-at-top-of-file | Module level import not at top of file